src/backend/InvenTree/order/models.py

```diff
@@ -696,6 +965,35 @@ def _action_cancel(self, *args, **kwargs):
                 content=InvenTreeNotificationBodies.OrderCanceled,
             )
 
+    def approval_allowed(self, user):
+        """Check that the given user is allowed to approve the order."""
+        active = getSetting('ENABLE_PURCHASE_ORDER_APPROVAL')
+        masters = getSetting('PURCHASE_ORDER_APPROVE_ALL_GROUP')
```

Comment:
Masters is an intresting word choice

Reply:
Well, I named it a master group in the settings, so. It just matched that choice 😅